### PR TITLE
Set up loom-js with webpack

### PR DESCRIPTION
**Pull request by andreipope**
Set up loom-js with webpack

### CODE REVIEW

**Comment by andreipope** [comment]
@enlight Hey, Vadim, please review when you get the chance. Thanks!

**Comment by enlight** [comment]
I don't deal with webpack unless it has me cornered, @redben can you review when you have a moment?

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -198,6 +198,7 @@ module.exports = {
         children: [
           ['/arch', 'arch'],
           ['/how-to-get-started', 'How to Get Started '],
+          ['how-to-loom-js-webpack','How to Set up loom-js with Webpack'],
```
[the remark this answers is not in the document]
Comment (andreipope) [reply docs/.vuepress/config.js]: thanks, it's fixed.

File: docs/en/how-to-loom-js-webpack.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
+---
+id: loom-js-webpack
+title: Set up loom-js with Webpack
```
[the remark this answers is not in the document]
Comment (andreipope) [reply docs/en/how-to-loom-js-webpack.md]: Yup, probably better.

File: docs/en/how-to-loom-js-webpack.md [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+sidebar_label: Set up loom-js with Webpack
```
[the remark this answers is not in the document]
Comment (andreipope) [reply docs/en/how-to-loom-js-webpack.md]: Fixed.

File: docs/en/how-to-loom-js-webpack.md [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+
+[Babel](https://babeljs.io/docs/en/index.html) is a tool that converts ECMAScript 2015+ code into JavaScript code that older browsers can run. With Babel, you can use the latest syntax without having to worry that a particular browser can't run it yet.
+
+1. Enter the following command to install Babel as a development dependency:
+
+```bash
+npm install --save-dev @babel/core @babel/cli @babel/preset-env babel-loader
+```
+
+2. To use Babel with Webpack, all what's lest to do is to add the following snippet to your `webpack.config.js`:
```
[the remark this answers is not in the document]
Comment (andreipope) [reply docs/en/how-to-loom-js-webpack.md]: Nice catch. Thanks!

File: docs/en/how-to-loom-js-webpack.md [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+
+## The Webpack development server
+
+1. Firt, we must install the Webpack development server with:
+
+```js
+npm install --save-dev webpack-dev-server
+```
+
+2. Now that the required package is installed, let's enable the Webpack development server by adding the following lines to the `module,exports` section of the `webpack.config.js` file:
```
[the remark this answers is not in the document]
Comment (andreipope) [reply docs/en/how-to-loom-js-webpack.md]: Aargh... a typo.

**Comment by andreipope** [comment]
@enlight It's updated.

File: docs/en/how-to-loom-js-webpack.md [hunk 6]
```diff
@@ -0,0 +1,365 @@
...
+For the scope of this tutorial, we'll write a simple demo that deploys a contract to our testnet, writes a value to the blockchain and then reads it.
+
+1. First, let's install a few dependencies:
+
+```bash
+npm install loom-js web3 solcjs-core
+```
+
+
+2. Next, edit your `src/index.js` to this:
```
Comment (enlight) [inline docs/en/how-to-loom-js-webpack.md]: What does `edit` mean here, does the file exist already and they have to replace the entire contents, or do they need to create a new one containing this code?

File: docs/en/how-to-loom-js-webpack.md [hunk 7]
```diff
@@ -0,0 +1,365 @@
...
+1. Because we want to run this in the browser, we must add the following snippet to the `module.exports` section of the `webpack.config.js` file:
+
+```js
+node: {
+ fs: 'empty',
+ child_process: 'empty'
+}
+```
+
+This way, we'll get rid of the compilation errors.
```
Comment (redben) [inline docs/en/how-to-loom-js-webpack.md]: fs and child_process are node modules, so we need to ignore them while building for the browser. This way, we'll get rid of....

Comment (andreipope) [reply docs/en/how-to-loom-js-webpack.md]: @redben Thanks! It's updated.

**Comment by andreipope** [comment]
@enlight @redben Just wanted to bump this your attention.  Is there anything holding this off from being approved and merged? Thanks!

File: docs/en/how-to-loom-js-webpack.md [hunk 8]
```diff
@@ -0,0 +1,373 @@
+---
+id: loom-js-webpack
+title: How to Set up loom-js with Webpack
```
Comment (enlight) [inline docs/en/how-to-loom-js-webpack.md]: -> `How to use loom-js with Webpack`

File: docs/en/how-to-loom-js-webpack.md [hunk 9]
```diff
@@ -0,0 +1,373 @@
...
+        }
+      },
+    ]
+  },
+}
+```
+
+## The Demo
+
+For the scope of this tutorial, we'll write a simple demo that deploys a contract to our testnet, writes a value to the blockchain and then reads it.
```
Comment (enlight) [inline docs/en/how-to-loom-js-webpack.md]: `For the scope of this tutorial` -> `In this tutorial`

**Comment by andreipope** [comment]
@enlight Thanks, it's updated.

**Pull request merged**